src/state_point.cpp

```diff
@@ -748,9 +748,12 @@ void read_source_bank(hid_t group_id, std::vector<Particle::Bank>& sites, bool d
   H5Tclose(banktype);
 }
 
-#ifdef DAGMC
 void write_unstructured_mesh_results() {
 
+#if !defined(LIBMESH) && !defined(DAGMC)
```

Comment:
Does this check need to be here, since you only call `write_unstructured_mesh_results` if either LIBMESH or DAGMC is defined?

Reply:
Nope. Safe to remove this. Thanks!